modules/nw-networkpolicy-about.adoc

```diff
@@ -64,7 +64,8 @@ spec:
 If the Ingress Controller is configured with `endpointPublishingStrategy: HostNetwork`, then the Ingress Controller pod runs on the host network.
 When running on the host network, the traffic from the Ingress Controller is assigned the `netid:0` Virtual Network ID (VNID).
 The `netid` for the namespace that is associated with the Ingress Operator is different, so the `matchLabel` in the `allow-from-openshift-ingress` network policy does not match traffic from the `default` Ingress Controller.
-Because the `default` namespace is assigned the `netid:0` VNID, you can allow traffic from the `default` Ingress Controller by labeling your `default` namespace with `network.openshift.io/policy-group: ingress`.
+Because the `default` namespace is assigned the `netid:0` VNID, you can allow traffic from the `default` Ingress Controller by labeling your `default` namespace with `network.openshift.io/policy-group: ingress` if you use OpenShift SDN.
+For the OVN-Kubernetes network provider plug-in, there is no supported configuration to allow traffic from Ingress Controller pods on the host network and use a network policy to deny traffic from all other sources.
```

Comment:
Is it possible to provide this as an IMPORTANT instead after the bullet is introduced?

```asciidoc
* Only allow connections from the {product-title} Ingress Controller:
+
To make a project allow only connections from the {product-title} Ingress Controller, add the following `NetworkPolicy` object:
```

Reply:
Agreed.  I had the same thought.  The remaining question in my mind is whether this doesn't apply in all cases, or only for the host network case.